src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3009,6 +3043,9 @@ ProduceBoundNode:
                                 End If
 
                                 argumentNamesLocationsBuilder.Add(id.GetLocation())
+                            ElseIf argumentNamesBuilder IsNot Nothing Then
+                                argumentNamesBuilder.Add(Nothing)
+                                argumentNamesLocationsBuilder.Add(Nothing)
```

Comment:
>argumentNamesLocationsBuilder.Add(Nothing) [](start = 32, length = 42)

 Same for ```Case SyntaxKind.RangeArgument```. I believe we should also disallow RangeArguments after named arguments in the parser, but the code will be more robust if we add an adjustment here, in case a syntax tree is built manually, etc.